pf/tfbridge/provider.go

```diff
@@ -148,7 +148,7 @@ func newProviderWithContext(ctx context.Context, info ProviderInfo,
 	}, nil
 }
 
-func newProviderServer(
+func NewProviderServer(
```

Comment:
This is exposed to facilitate testing as tests live in another go module to not pollute the main module with dependencies of test resource providers such as the random provider. I think this may be OK to expose but we owe a doc string later on.